Android/firebase_dependencies.gradle

```diff
@@ -37,7 +37,7 @@ def firebaseDependenciesMap = [
                  'com.google.flatbuffers:flatbuffers-java:1.12.0',
                  'com.google.android.gms:play-services-base:17.6.0'],
   'performance' : ['com.google.firebase:firebase-perf:20.0.2'],
-  'remote_config' : ['com.google.firebase:firebase-config:21.0.0',
+  'remote_config' : ['com.google.firebase:firebase-config:21.0.1',
```

Comment:
Is this change intentional?

Reply:
Yes, this is part of the version update that we want to release.